content/tools.markdown

```diff
@@ -63,10 +63,11 @@ Manage zones for on-premise and self-hosted CoreDNS instances with DNSimple.
 
 - [Import tool in Ruby](https://github.com/aetrion/dnsimple-importer)
 - [Dynamic IP Updater in Bash](/ddns) (official)
-- [Dynamic IP Updater in Bash](http://jasonseifer.com/2011/04/04/auto-update-ip-dnsimple) (original by [Jason Seifer](http://jasonseifer.com)
-- [Dynamic IP Updater in Ruby](https://github.com/hamstarr/dyn-dnsimple) by [hamstarr](https://github.com/hamstarr)
-- [Dynamic IP Updater in Ruby](https://github.com/bjoernalbers/dynsimple) by [bjoernalbers](https://github.com/bjoernalbers)
+- [Dynamic IP Updater in Bash](http://jasonseifer.com/2011/04/04/auto-update-ip-dnsimple) original by [Jason Seifer](http://jasonseifer.com)
 - [Dynamic IP Updater in Node.js](https://github.com/jcarley/node-dns) by [jcarley](https://github.com/jcarley)
+- [Dynamic IP Updater in Python](https://github.com/oleduc/dnsimple-dynamic-dns) by [oleduc](https://github.com/oleduc)
+- [Dynamic IP Updater in Ruby](https://github.com/bjoernalbers/dynsimple) by [bjoernalbers](https://github.com/bjoernalbers)
```

Comment:
This project link seems to be broken

Reply:
Thank you for flagging it. I also found other links that were dead and have been removed:

```markdown
- [Dynamic IP Updater in Bash](http://jasonseifer.com/2011/04/04/auto-update-ip-dnsimple) original by [Jason Seifer](http://jasonseifer.com)
- [Import tool in Ruby](https://github.com/aetrion/dnsimple-importer)
```